core/Cargo.toml

```
@@ -9,7 +9,7 @@ generic-ec = { git = "https://github.com/dfns-labs/generic-ec", branch = "m", de
 
 rand_core = { version = "0.6", default-features = false }
 serde = "1"
-hex = { version = "0.4", default-features = false, features = ["alloc", "serde"]}
+base64 = "0.21.4"
```

Generally, you don't need to specify a patch version

```suggestion
base64 = "0.21"
```